lib/index.js

```diff
@@ -13,5 +13,38 @@ module.exports = (dirname, manifestExtras) => {
 
     const manifest = Manifest.concat(manifestExtras || []);
 
-    return Haute(dirname, manifest);
+    return (server, options, next) => {
+
+        if (typeof options === 'function' ||
+           typeof next !== 'undefined') {
+
+            // callback style
+            if (typeof next !== 'undefined') {
+                return Haute(dirname, manifest)(server, options, next);
+            }
+
+            next = options;
+            return Haute(dirname, manifest)(server, null, next);
+        }
+
+        return new Promise((resolve, reject) => {
+
+            const cb = function (err) {
+
+                if (err) {
+                    reject(err);
```

Comment:
Coveralls is asking for coverage for these lines, that's why I was trying to cause Haute to error. What's the right way to go about this?